aws/terminator/application_services.py

```diff
@@ -150,6 +150,40 @@ def terminate(self):
         self.client.delete_file_system(FileSystemId=self.id)
 
 
+class KinesisStream(Terminator):
+    @staticmethod
+    def create(credentials):
+        def paginate_streams(client):
+            names = client.get_paginator('list_streams').paginate().build_full_result()['StreamNames']
```

Comment:
How many streams do our integration tests create? I agree the default limit seems too low. I'd probably set the limit to a much higher value, perhaps 50 or 100.

Reply:
@mattclay today, zero because we have no integration tests. I'm waiting to merge this PR here https://github.com/ansible/ansible/pull/62114 though which will create 1 and tear it down.

Reply:
@mattclay @jillr @s-hertel I've increased the default page size to 100 per @mattclay's comment.